spec/acceptance/nodesets/default.yml

```diff
@@ -0,0 +1,9 @@
+HOSTS:
+  centos8:
+    platform: el-8-x86_64
+    hypervisor: docker
+    image: amd64/centos:centos8
```

Comment:
This appears to be the only difference. Perhaps https://github.com/voxpupuli/puppet_metadata/blob/master/lib/puppet_metadata/github_actions.rb should be modified to emit this this specific image but it may be tricky to do so cleanly.